### DISCUSSION TITLE
Webinar: Web Unification - Optimizing Your Access to NASA Earth Science Data

### DISCUSSION THREAD

**Discussion by tjstogoski** [earthlab-education/Earth-Analytics-AY24#117] [General]
I learned about this webinar and thought it might be of interest to others! Registration is in the link below.

From: [NASA Earth Data](https://www.earthdata.nasa.gov/learn/webinars-and-tutorials/web-unification-09-16-2024)

"Join us Sep 16 at 2 p.m., EDT [-04:00 UTC], to learn about updates taking place on the public facing Earthdata website in October 2024, including a preview of the website prototype and a review of the project timeline. 

The Earth Science Data Systems ([ESDS](https://www.earthdata.nasa.gov/esds)) Program has managed the full life cycle of NASA Earth science data since 1994. As technology has evolved, so too has the management of these data. We are constantly introducing new ways to improve how we handle data while also responding to new policies. This is the genesis of the Web Unification Project.

In response to requirements outlined in the [21st Century Integrated Digital Experience Act](https://www.congress.gov/bill/115th-congress/house-bill/5759/text), NASA launched the Web Modernization project to consolidate all of the agency's websites (there are thousands) into nasa.gov or science.nasa.gov. ESDS followed suit and launched the Web Unification Project in December 2021 to migrate all ESDS-funded web properties to earthdata.nasa.gov by the end of 2026.

This webinar will focus on updates that will take place on the public facing [Earthdata](https://earthdata.nasa.gov/) website in October 2024. Webinar highlights include a preview of the new website prototype, background on which websites are consolidating into Earthdata and a review of the timeline for this work, and information about where to get help in finding what you are looking for during this transition."

**Comment by eculler** [comment]
Awesome @tjstogoski ! We will be taking advantage of some of the great work NASA and the LPDAAC have done to make their data more available, including the AppEEARS and earthaccess APIs. It's truly a huge advancement in data accessibility they've made in just a few years.
